clients/fides-js/src/lib/consent-types.ts

```diff
@@ -398,7 +400,11 @@ export enum ButtonType {
 }
 
 export enum ConsentMethod {
-  button = "button",
+  button = "button", // deprecated- keeping for backwards-compatibility
+  reject = "reject",
+  accept = "accept",
+  save = "save",
+  dismiss = "dismiss",
```

Comment:
it is odd that this is lower case when the admin-ui type is uppercase... we don't need to fix that now, but normally our enums are capitalized!

Reply:
yeah true. A larger cleanup is needed, where a lot of enums in this file are lowercase too...